packages/design-system-react/src/components/button-secondary/index.ts

```diff
@@ -0,0 +1,3 @@
+export { ButtonSecondary } from './ButtonSecondary';
+export type { ButtonSecondaryProps } from './ButtonSecondary.types';
+export { ButtonBaseSize as ButtonSecondarySize } from '../button-base';
```

Comment:
Here we are using an alias for `ButtonBaseSize` this ensures a source of truth and alignment with the component